spec/spec.md

```diff
@@ -328,15 +328,20 @@ where:
 ```
 {
     “auth_token”: “<auth_token>”,
-    “addresses”: [“<address>", ...]
+    “labelled_addresses”: [
+        {“address”: “<address>", “label”: “<label>”},
+        ...
+    ],
     “wallet_uri_base”: “<wallet_uri_base>”,
 }
 ```
 
 where:
 
 - `auth_token`: an opaque string representing a unique identifying token issued by the wallet endpoint to the dapp endpoint. The format and contents are an implementation detail of the wallet endpoint. The dapp endpoint can use this on future connections to `reauthorize` access to [privileged methods](#privileged-methods).
-- `addresses`: one or more base64-encoded addresses, for the accounts to which this auth token corresponds
+- `labelled_addresses`: one or more value objects that represent the accounts to which this auth token corresponds. These objects hold the following properties:
+  - `address`: a base64-encoded address
+  - `label`: (optional) a human-readable string that describes the address. Wallets that allow their users to label their accounts may choose to return those labels here to enhance the user experience in the target dapp.
```

Comment:
NIT: I've been using the terms "wallet endpoints" and "dapp endpoints", rather than just "wallet" and "dapp", to avoid confusing wallet-applications with wallet-accounts